packages/govuk-frontend/src/govuk/helpers/_focused.scss

```diff
@@ -26,3 +26,24 @@
   // box-shadow is applied to each fragment independently.
   box-decoration-break: clone;
 }
+
+/// Focused image
+///
+/// Provides an outline to clearly indicate when the target element is focused.
+/// Used for non-text content contained within links.
+///
+/// @access public
+
+@mixin govuk-focused-image {
```

Comment:
Prompted by Colin's comment, wondering if this should be made slightly more generic as it could be applied to non-images? Maybe something like `govuk-focused-box`?